specification/2.0/appendix/performance-metrics/README.md

```diff
@@ -0,0 +1,107 @@
+# Runtime Performance Metrics 
+
+The goal of these metrics is to assess the runtime performance needs of glTF assets.  
+By runtime we mean the cpu/gpu, bandwidth and memory resources needed to render the model in realtime.  
+These performance metrics are divided in two major categories - `complexity` and `memory`
+
+## Performance Metric Categories
+
+### Complexity  
+
+The `complexity`category provides metrics to calculate cpu/gpu and memory bandwidth target requirements - larger numbers will mean a more powerful target device is needed.  
+These metrics do not provide a mapping to target device performance, this mapping must be done by users of the metrics.  
+
+### Screen space coverage
+
+The metrics do not include screen space coverage, it is advised that clients shall know how large the render area will be before requesting a model (from the backend)  
+It is up to the backend to take screen coverage into account when providing a suitable model to the client, for instance by taking width + height as parameters.  
```

Comment:
I think I might rephrase this in terms that don't assume a backend server exists. For example, the models may be stored statically (disk, CDN) and fetched by a client, where the client considers screen coverage when deciding how much it is willing to pay in performance metrics. For example:

> The metrics do not include screen space coverage. However, implementations should generally consider the size of the render area in addition to performance metrics of the model. A higher performance budget may be acceptable when screen space coverage is higher.

Reply:
I wanted to highlight the complexity that server-reference brings in, as well.  But Don's phrasing captures my thinking.  
Only comment I would add is... given that we are talking about Spatial stuff, 'screen' might not be the accurate word choice.  I don't have a concrete alternative, but something to do with 'volume' may be?

Reply:
My intention here was to determine the screen bounds of the asset, as used in this extension:
https://github.com/KhronosGroup/glTF/blob/master/extensions/2.0/Vendor/FB_geometry_metadata/README.md
My understanding is that if we had a fairly "detailed" asset with lots of vertices but it's final screen bounds turned out to be small say: 5 cm cube, The GPU would save compute in the final rasterization phase.

Reply:
Two components of this:

1. Number of pixels to be rendered is a multiplier of GPU cost, particularly with more expensive PBR materials. An expensive material is more costly to render the more pixels it shades.
2. A client may _want_ to spend more of its performance budget on an asset covering more of the screen/viewport/space.

I think (2) is the more relevant piece here. (1) comes into play when choosing materials for background environments (high coverage, low detail) vs. hero assets (low coverage, high detail) as an artist. These are general graphics best practices and tradeoffs, I'm not sure how much we should specify.

Reply:
I see what you mean about spatial extent being a third component, although it mostly affects performance insomuch as it affects the number of pixels shaded, and ultimately only the application can know the relationship between spatial extent and screen space.

Reply:
I agree with @donmccurdy - I have updated the screen space section according to his comments.

Reply:
> My intention here was to determine the screen bounds of the asset, as used in this extension:

As far as I can tell the bounds in that extension are boundingboxes.
These can be used, for instance, to choose scene (LoD) depending on the runtime transform of the model.
Whereas in our case I think we should take a 'worst case' scenario approach.
Meaning that if the framebuffer (render area) is 500 * 500 pixels then we should assume that most of these pixels are drawn (maybe 70% or something) - but this is up to the client.

I will try to find a wording for this.